### PR TITLE
Fix three Toast bugs

### PR DESCRIPTION
**Pull request by holder-josh**
cc @OliverJAsh,  just a few minor bug fixes for toast:
- Loading button no longer shows when it shouldn't (was related to adding the loading class when there are no new blocks)
- IE doesn't support window.scrollY
- Remove unnecessary margin on the toast button

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/ui/newAutoupdate.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ define([
         };
 
         var scrolledPastTopBlock = function () {
-            return $liveblogBody.offset().top < window.scrollY;
+            return $liveblogBody.offset().top < window.pageYOffset;
         };
         var isLivePage = !(window.location.href.search('[?&]page=') !== -1);
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/ui/newAutoupdate.js]: no need the double negative
`var isLivePage = (window.location.href.search('[?&]page=') === -1);`

**Comment by johnduffell** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/ui/newAutoupdate.js (hunk 1 above)
Comment (piuccio) [inline static/src/javascripts/projects/common/modules/ui/newAutoupdate.js]: isn't it `window.location.search` ? https://developer.mozilla.org/en-US/docs/Web/API/Location

Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/ui/newAutoupdate.js]: good point,t hat's what I used and it seems(!) to work
https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/enhanced/liveblog.js#L156

Comment (Jholder112233) [inline static/src/javascripts/projects/common/modules/ui/newAutoupdate.js]: Yep, error there that needs fixing. @OliverJAsh, do you mind fixing that before shipping this? 

Thanks very much.

**Comment by michaelwmcnamara** [comment]
[ ] - Loading button showing when it shouldn't - Fixed
[ ] - Formatting on toast popup - Fixed
[ ] - Toast notifications not working on IE - not fixed - no toast is displayed in IE
[ ] - check all on later pages of liveblogs - still to come

File: static/src/javascripts/projects/common/modules/ui/newAutoupdate.js (hunk 1 above)
Comment (OliverJAsh) [inline static/src/javascripts/projects/common/modules/ui/newAutoupdate.js]: I think you have done it correctly, actually. `window.location.href.search` is a function on the `String` prototype, which is what you want. `window.location.search` is a string of query params.

Question is, how well supported is `window.location.href.search`…?

Comment (OliverJAsh) [inline static/src/javascripts/projects/common/modules/ui/newAutoupdate.js]: Will change to `window.location.search.indexOf` as I think that makes sense

Comment (piuccio) [inline static/src/javascripts/projects/common/modules/ui/newAutoupdate.js]: [String.search](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/search), I like that, just a bit confusing when you read it

**Comment by OliverJAsh** [comment]
Seems to be working on all the IEs when opted into this test as well as `RemoveStickyNav=new`. Test URL: http://localhost:9000/media/live/2016/feb/05/julian-assange-is-being-arbitrarily-detained-un-panel-finds-live-updates#ab-LiveblogToast=toast,RemoveStickyNav=new

**Comment by OliverJAsh** [comment]
This is ready to go. I'll ship it when I'm done with sticky :smile:

**Pull request merged**